index.md

```diff
@@ -5,18 +5,17 @@ permalink: index.html  # Is the only page that doesn't follow the pattern /:path
 ---
 Apptainer (formerly known as Singularity) is a free and open-source container platform that allows you to create and run applications in isolated images (also called "containers") in a simple, portable, fast, and secure manner. It performs operating system level virtualization known as containerization. Many container platforms are available, but Apptainer is designed to bring containers and reproducibility to the scientific community and High-Performance Computing (HPC) use cases. Using Apptainer, developers can work in reproducible environments of their choice and design, and these complete environments can be easily copied and executed on other platforms.
 
-An introduction to Singularity/Apptainer, its motivations and applications in HEP.
+This is An introduction to Singularity/Apptainer, its motivations and applications in HEP.
```

Comment:
A minor thing, but here capital 'A' must be lowercase.

Reply:
done